app/react/portainer/environments/EdgeAutoCreateScriptView/AutomaticEdgeEnvCreation/AutomaticEdgeEnvCreation.tsx

```diff
@@ -23,6 +23,7 @@ const commands = {
     commandsTabs.k8sLinux,
     commandsTabs.swarmLinux,
     commandsTabs.standaloneLinux,
+    commandsTabs.podmanLinux,
```

Comment:
Guard this insert with `useFeatureFlag(FeatureId.PODMAN)` or a custom hook (this guard is used in many places)

```ts
function useIsPodmanEnabled() {
  const { data } = useFeatureFlag(FeatureId.PODMAN);
  return !!data;
}
```

The benefit of using a custom hook is that you won't have to deal with `boolean | undefined` in other parts of the app

Reply:
Updated, nice spot to guard this.

The plan is to use the feature flag so I can independently merge develop PRs without any effect on behaviour, then remove it when all subtasks are complete and testing passes.